assets/php/api/class-wc-rest-blocks-products-controller-v2.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * REST API Products controller customized for Products Block.
+ *
+ * Handles requests to the /products endpoint.
+ *
+ * @internal These overrides need rolling into core when releasing a new version.
+ *
+ * @package WooCommerce\Blocks\Products\Rest\Controller
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST API Products controller class.
+ *
+ * @package WooCommerce/API
+ */
+class WC_REST_Blocks_Products_Controller_V2 extends WC_REST_Blocks_Products_Controller {
```

Comment:
So this class is called `_V2` but you haven't actually bumped the API version, is that right?

Reply:
Correct. I added the suffix to avoid naming collision. I'm just modifying (adding to) the v1 endpoint really.